docs/UserGuide.md

```diff
@@ -299,6 +348,28 @@ Figure 6.2.3.1. Example of edit budget command
 
 #### 6.2.4. Finding budgets: `find`
 
+You can use this command to **quickly find your budgets**.
+
+Use this command to filter through your budgets by using keywords. NUSave will display all budgets whose title matches 
+the entered keyword/key phrase. 
+
+ > ⚠ **You should not use this command if main page is empty**. This is because there are no budgets created yet. 
+ > You can only find a budget if there are already existing budgets!
+
+Format: `find WORD`
```

Comment:
This is fine but inconsistent with what I used, I used find NAME for my format instead. Let's settle on one to keep things consistent! 👍

Reply:
got it!